baseplate/lib/thrift_pool.py

```diff
@@ -114,6 +116,8 @@ def thrift_pool_from_config(
         kwargs.setdefault("max_connection_attempts", options.max_connection_attempts)
     if options.max_retries is not None:
         raise Exception("max_retries was renamed to max_connection_attempts")
+    if options.queue is not None:
+        kwargs.setdefault("queue", "lifo")
```

Comment:
What about defining the default in the config parser and pulling it from there for the kwargs update?
`kwargs.setdefault("queue", options.queue)`